tardis/tests/tests_slow/test_w7.py

```diff
@@ -1,81 +1,15 @@
 import os
```

Comment:
I think this will eventually become a common class for running multiple setups, and putting it in w7 dir might not suite it. When more setups join in, we can surely change the filename and class name to something more generalized. Thoughts ?